.github/workflows/windows-build.yaml

```diff
@@ -0,0 +1,106 @@
+name: Windows
+
+on:
+  push:
+    branches:
+     - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  windows-staging:
+    name: Windows staging build
+    runs-on: windows-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+
+      - name: Install Qt
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git clone https://github.com/bakulf/qt_static_windows
+          cd qt_static_windows
+          cat x* > qt_static.tar.bz2
+          tar xf qt_static.tar.bz2
+          tar xf msm.tar.gz
+          mkdir /c/MozillaVPNBuild
+          cp -r * /c/MozillaVPNBuild
+          cd ..
+          cp /c/MozillaVPNBuild/bin/libssl-1_1-x64.dll .
+          cp /c/MozillaVPNBuild/bin/libcrypto-1_1-x64.dll .
+          cp /c/MozillaVPNBuild/bin/libEGL.dll .
+          cp /c/MozillaVPNBuild/bin/libGLESv2.dll .
+          cp /c/MozillaVPNBuild/*.msm .
+
+      - name: Adding msbuild to PATH
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Compilation script
+        shell: bash
+        run: |
+          export PATH=/c/MozillaVPNBuild/bin:$PATH
+          ./scripts/windows_compile.bat
+
+      - name: Upload app
+        uses: actions/upload-artifact@v2
+        with:
+            name: staging
+            path: windows/installer/x64/MozillaVPN.msi
+
+  windows-production:
+    name: Windows production build
+    runs-on: windows-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+
+      - name: Install Qt
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git clone https://github.com/bakulf/qt_static_windows
```

Comment:
I have this repo to host the pre-compiled static QT. When we will have a proper CI, this repo will be removed.